pkg/reconciler/v1alpha1/autoscaling/autoscaling.go

```diff
@@ -169,6 +174,53 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 
 func (c *Reconciler) reconcile(ctx context.Context, key string, kpa *kpa.PodAutoscaler) error {
 	logger := logging.FromContext(ctx)
+	logger.Debugf("PodAutoscaler class %q", kpa.Class())
+
+	// Create or update the underlying autoscaler implementation.
+	// Delete unused autoscaler implementations--there can be only one!
+	switch kpa.Class() {
+	case "kpa":
+		err := c.deleteHpa(ctx, key)
+		if err != nil {
+			return err
+		}
+		metric, err := c.upsertKpa(ctx, key, kpa)
+		if err != nil {
+			return err
+		}
+		// Right-size the ScaleTargetRef and update status.
+		return c.reconcileMetric(ctx, kpa, metric)
+	case "hpa":
```

Comment:
What will happen to the metrics sent by the pods on this path? Are they dropped immediately or is there some buffer we might be filling in the long-run?

Reply:
It just drops them on the floor: https://github.com/knative/serving/blob/81d9d5a722407ebaa357801dc7b7729b73095d6a/pkg/autoscaler/multiscaler.go#L239 (desired behavior).

Reply:
Ah, nice!